pkg/alertmanager/multitenant_test.go

```diff
@@ -2520,10 +2520,20 @@ func TestComputeConfig(t *testing.T) {
 	cfg2.GrafanaAlertmanagerTenantSuffix = "-grafana"
 	amWithSuffix := setupSingleMultitenantAlertmanager(t, cfg2, store, nil, featurecontrol.NoopFlags{}, log.NewNopLogger(), reg2)
 
-	tenantReceivingRequests := "receiving-1-grafana"
-	amWithSuffix.lastRequestTime.Store(tenantReceivingRequests, time.Now().Unix())
-	tenantReceivingRequestsExpired := "receiving-2-grafana"
```

Comment:
I'm removing the `-grafana` suffix from the test tenant to test the behavior of Mimir tenants (no suffix) and Grafana tenants separately. This becomes clearer in line 3427.